app/models/account.rb

```diff
@@ -120,7 +120,7 @@ def groups
   end
 
   def add_group(owner)
-    with_valid_account_type('user') { set_parent(owner) }
+    with_valid_account_type('user') { add_parent(owner) }
```

Comment:
Am I reading this correctly that these two places are the only ones where we want might want to have multiple children/parents?